stytch/client.go

```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 )
 
-const apiVersion = "1.1.3"
+const apiVersion = "1.2.0"
```

Comment:
If we're doing semver, I think removing exported names would make this a major version bump.
```suggestion
const apiVersion = "2.0.0"
```